common/beerocks/bwl/dummy/sta_wlan_hal_dummy.cpp

```diff
@@ -16,9 +16,10 @@
 namespace bwl {
 namespace dummy {
 
-sta_wlan_hal_dummy::sta_wlan_hal_dummy(const std::string &iface_name, hal_event_cb_t callback)
+sta_wlan_hal_dummy::sta_wlan_hal_dummy(const std::string &iface_name, hal_event_cb_t callback,
+                                       const bwl::hal_conf_t &hal_conf)
     : base_wlan_hal(bwl::HALType::Station, iface_name, IfaceType::Intel, callback, {}),
```

Comment:
`base_wlan_hal ctor` also accept `hal_conf` as one of arguments:
```
    base_wlan_hal(HALType type, std::string iface_name, IfaceType iface_type,
                  hal_event_cb_t callback, hal_conf_t hal_conf = {});
```
So pass actual `hal_conf`into `base_wlan_hal` constructor instead of default `{}`

Reply:
Fixed